test/integration/targets/filesystem/defaults/main.yml

```diff
@@ -15,7 +15,7 @@ tested_filesystems:
   ext3: {fssize: 10, grow: True}
   ext2: {fssize: 10, grow: True}
   xfs: {fssize: 20, grow: False}  # grow requires a mounted filesystem
-  btrfs: {fssize: 100, grow: False}  # grow not implemented
+  btrfs: {fssize: 150, grow: False}  # grow not implemented
```

Comment:
Related comment line 7 wasn't updated.

Reply:
Thanks for the pickup, has been updated with https://github.com/ansible/ansible/pull/47260